server.js

```diff
@@ -4,14 +4,23 @@ var express = require('express'),
 	server = http.createServer(app);
 
 app.use('/', express.static(__dirname + '/public'));
-app.use('/lib', express.static(__dirname + '/node_modules/jxml/public/lib'));
-app.use('/lib', express.static(__dirname + '/node_modules/jxml/node_modules/systemjs/dist'));
-app.use('/lib', express.static(__dirname + '/node_modules/jxml/node_modules/systemjs/node_modules/es6-module-loader/dist'));
-app.use('/lib', express.static(__dirname + '/node_modules/jxml/node_modules/systemjs/node_modules/es6-module-loader/node_modules/traceur/bin'));
+app.use('/lib', express.static(resolve('jxml/public/lib/jxml.js')));
+app.use('/lib', express.static(resolve('systemjs/dist/system.js')));
+app.use('/lib', express.static(resolve('es6-module-loader/dist/es6-module-loader.js')));
+app.use('/lib', express.static(resolve('traceur/bin/traceur.js')));
 app.use('/lib/jx', express.static(__dirname + '/jx'));
-app.use('/lib/jx', express.static(__dirname + '/node_modules/jxml/jx'));
-app.use('/lib/jx/data', express.static(__dirname + '/node_modules/datanode'));
-app.use('/lib/datanode', express.static(__dirname + '/node_modules/datanode'));
+app.use('/lib/jx', express.static(resolve('jxml/jx/JXML')));
+app.use('/lib/jx', express.static(resolve('datanode')));
```

Comment:
actually, it should be just `/lib` since `resolve('datanode')` returns the directory with `jx/data`